### PR TITLE
Changed method calls that dont exist in D9

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- D9 methods don't exist. Switch them out

# Need Review By (Date)
- ASAP

# Urgency
- Critical

# Steps to Test
https://app.circleci.com/pipelines/github/SU-HKKU/cardinal_service_profile/1558/workflows/0b590491-ff4f-42ac-b424-154763629287/jobs/3353

image

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: stanford_migrate.module [hunk 1]
```diff
@@ -92,7 +92,7 @@ function stanford_migrate_entity_delete(EntityInterface $entity) {
       strpos($destination['plugin'], 'entity:') !== FALSE ||
       strpos($destination['plugin'], 'entity_reference_revisions:') !== FALSE
     ) {
-      list(, $type) = explode(':', $destination['plugin']);
+      [, $type] = explode(':', $destination['plugin']);
```
Comment (imonroe) [inline stanford_migrate.module:95]: This syntax hurts my brain.

**Review by imonroe — APPROVED** [review]
Ok by me.

File: stanford_migrate.module (hunk 1 above)
Comment (jbickar) [reply stanford_migrate.module:95]: Agreed. Some code comments are in order here.

Comment (pookmish) [reply stanford_migrate.module:95]: yeah i hate that phpstorm wraps short array declarations and list declarations into the same setting. So it does the conversion without me even noticing it.
image

**Pull request merged**